dashboards/geospatial/bigquery_geospatial_visualization.ipynb

```diff
@@ -0,0 +1,955 @@
+        "##(Optional) Enable Google Maps Javascript API"
+      ],
+      "metadata": {
+        "id": "OkNoiTixjt9F"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!gcloud services enable \\\n",
+        "    --project $GCP_PROJECT_ID \\\n",
+        "    \"maps-backend.googleapis.com\""
+      ],
+      "metadata": {
+        "id": "LK4gaoU0INDv"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Install Packages"
+      ],
+      "metadata": {
+        "id": "s7_S0DklRQip"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install pydeck>=0.9 h3>=4.2"
+      ],
+      "metadata": {
+        "id": "DpeceuOPRSP_"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "##Import libraries"
+      ],
+      "metadata": {
+        "id": "ycaYw0I-j8P1"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import branca\n",
+        "import geemap.deck as gmdk\n",
+        "import h3\n",
+        "import pydeck as pdk\n",
+        "import geopandas as gpd\n",
+        "import shapely"
+      ],
+      "metadata": {
+        "id": "kSrLzumZq2Wj"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Enable interactive tables for `pandas` DataFrames in Colab.\n",
+        "\n",
+        "> See [this article](https://colab.google/articles/alive) for more details."
+      ],
+      "metadata": {
+        "id": "iB0Hd0JBNWHC"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Enable displaying pandas data frames as interactive tables by default\n",
+        "from google.colab import data_table\n",
+        "data_table.enable_dataframe_formatter()"
+      ],
+      "metadata": {
+        "id": "fa0BY4cqpEUt"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Shared Routines\n",
+        "\n",
+        "We create `display_pydeck_map` here to render charts using `pydeck`.  This\n",
+        "involves creating a `pydeck.Map` instance and then adding layers (`pydeck.Layer`) to the map instance."
+      ],
+      "metadata": {
+        "id": "wathhRxeKFA0"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "MAP_PROVIDER_GOOGLE = pdk.bindings.base_map_provider.BaseMapProvider.GOOGLE_MAPS.value\n",
+        "\n",
+        "# Shared routine for rendering layers on a map using geemap.deck\n",
+        "def display_pydeck_map(layers, view_state, **kwargs):\n",
+        "  deck_kwargs = kwargs.copy()\n",
+        "\n",
+        "  # use Google Maps as the base map only if the API key is provided\n",
+        "  if GMP_API_KEY:\n",
+        "    deck_kwargs.update({\n",
```

Comment:
Add `"ee_initialize": False,` to the deck_kwargs to prevent initialization of Google Earth Engine API

Reply:
I moved it to directly to the `gmkd.Map` constructor call instead.  `deck_kwargs` can override it.

ie. If the reader clones the tutorial and does want to use EarthEngine for their own purposes, they can specify it in the `kwargs` for `display_pydeck_map`, they can pass in `ee_initialize=True` to override it.